src/components/Buttons.js

```diff
@@ -3,10 +3,11 @@ import '../index.css';
 
 export default function Buttons({
   // eslint-disable-next-line react/prop-types
```

Comment:
- [OPTIONAL] I would advise you to remove the `eslint-disable` statement. `Eslint` helps us to write industry-standard, clean and readable codes. Using `eslint-disable` statement defeats this purpose, kindly remove it. 👍 💻